pkg/executor/reaper/reaper.go

```diff
@@ -238,8 +238,7 @@ func CleanupRoleBindings(ctx context.Context, logger *zap.Logger, client kuberne
 				// so now we need to look for the objects in default namespace.
 				saNs := subj.Namespace
 				isInReservedNS := false
-				if subj.Namespace == functionNs ||
-					subj.Namespace == envBuilderNs {
```

Comment:
Any reason to remove envBuilderNs ?

Reply:
no nothing specific, I forgot to revert my changes. Now it's done